frontend/src/App.js

```diff
@@ -94,6 +105,7 @@ class App extends Component {
                             <ul>
                               <li><ActiveLink to="/dashboard" label="Dashboard" /></li>
                               <li><ActiveLink to={`/users/${profile.id}`} label="Public Profile" /></li>
+                              <li><ActiveLink to={`/edit/${profile.id}`} label="Edit Profile" /></li>
```

Comment:
Note: We should probably rethink the router here and make this:
1. An authenticated only route
2. Part of the dashboard page